### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by RickWinter**
Add contributing notes to the samples readme

### CODE REVIEW

File: samples/README.md [hunk 1]
```diff
@@ -41,3 +41,21 @@ This section lists how-to guides for the most commonly used APIs and most common
...
+This project welcomes contributions and suggestions. Most contributions require you to agree to a [Contributor License Agreement (CLA)][cla] declaring that you have the right to, and actually do, grant us the rights to use your contribution.
+
+If you'd like to contribute to this library, please read the [contributing guide] [contributing_guide] to learn more about how to build and test the code.
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For more information, see the [Code of Conduct FAQ][coc_faq] or contact [opencode@microsoft.com][coc_contact] with any additional questions or comments.
+
+<!-- LINKS -->
+[contributing_guide]: https://github.com/Azure/azure-sdk-for-go/blob/main/CONTRIBUTING.md
```
Comment (RickWinter) [inline samples/README.md:56]:
```suggestion
[contributing_guide]: https://github.com/Azure/azure-sdk-for-cpp/blob/main/CONTRIBUTING.md
```

**Review by LarryOsterman — APPROVED** [review]

File: samples/README.md [hunk 2]
```diff
@@ -41,3 +41,21 @@ This section lists how-to guides for the most commonly used APIs and most common
 #### General How-To Guides
 
 - How to configure, access, and analyze **logging** information (TODO)
+
+## Contributing
+
+This project welcomes contributions and suggestions. Most contributions require you to agree to a [Contributor License Agreement (CLA)][cla] declaring that you have the right to, and actually do, grant us the rights to use your contribution.
+
+If you'd like to contribute to this library, please read the [contributing guide] [contributing_guide] to learn more about how to build and test the code.
```
Comment (ahsonkhan) [inline samples/README.md:49]:
```suggestion
If you'd like to contribute to this library, please read the [contributing guide][contributing_guide] to learn more about how to build and test the code.
```

Comment (ahsonkhan) [reply samples/README.md:49]: Otherwise, it loooks like this:
image

Comment (RickWinter) [reply samples/README.md]: Good catch.  This came verbatim from compliance team.  I made the fix

**Pull request merged**